ansible_mitogen/transport_config.py

```diff
@@ -384,6 +390,12 @@ def ansible_ssh_timeout(self):
             self.timeout()
         )
 
+    def ssh_host_key_checking(self):
+        if self._connection.get_task_var('ansible_ssh_host_key_checking') is not None:
```

Comment:
Please simplify this to:
```
return self._connection.get_task_var('ansible_ssh_host_key_checking') or self._connection.get_task_var('ansible_host_key_checking')
```
which saves multiple calls to `get_task_var('ansible_ssh_host_key_checking')`

Reply:
@s1113950 I tried your proposed change. The problem is, that if
* `ansible_ssh_host_key_checking` is set to false and
* `ansible_host_key_checking` is not set

the `or` version would return `None` instead of `false`.

Reply:
ahh! Good catch. Then instead of calling `self._connection.get_task_var('ansible_ssh_host_key_checking')` twice, can you make it a variable and use it in your if block? Reason being `get_task_var` can get expensive depending on how much walking-the-stack needs to be done

Reply:
Good point. Tanks.